packages/devextreme-cli/src/templates-ts/react/application/src/app-routes.tsx

```diff
@@ -0,0 +1,24 @@
+import { HomePage, TasksPage, ProfilePage } from './pages';
+import { withNavigationWatcher } from './contexts/navigation';
+
+const routes = [
```

Comment:
The same for routes. Maybe add a route type?

Reply:
The same. Typescript understand this type.